<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogs</class>
 <widget class="QDialog" name="DialogLogs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Azioni utenti</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>710</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>710</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>710</width>
     <height>320</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>710</width>
     <height>320</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>710</width>
     <height>420</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>159</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>318</number>
   </attribute>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Utente</string>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Azione</string>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelFrom">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Da</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTo">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>7</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>30</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Utente</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarFrom">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>4</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarTo">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>4</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btFind">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>30</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
